src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-lenght-29">
+      <div class="calendar__day calendar__day--start-sun"></div>
```

Comment:
this modifier as `calendar--start-day--sun` should use for the parent (.calendar), not for the child (.calendar__day)